pages/report-repair/[route].js

```diff
@@ -252,6 +315,10 @@ export async function getStaticPaths() {
     {params: { route: 'repair-location'} },
     {params: { route: 'smell-gas'} },
     {params: { route: 'repair-kitchen-problems'} },
+    {params: { route: 'repair-bathroom-problems'} },
+    {params: { route: 'repair-bedroom-problems'} },
+    {params: { route: 'repair-living-areas-problems'} },
+    {params: { route: 'wall-problems'} },
```

Comment:
After renaming in other comment, this would also need renaming
```suggestion
    {params: { route: 'wall-floor-ceiling-problems'} },
```